crypto/asn1/a_int.c

```diff
@@ -260,12 +260,16 @@ static int asn1_get_int64(int64_t *pr, const unsigned char *b, size_t blen,
         return 0;
     if (neg) {
         if (r <= INT64_MAX) {
-            /* Most significant bit is guaranteed to be clear, negation
-             * is guaranteed to be meaningful in platform-neutral sense. */
+            /*
+             * Most significant bit is guaranteed to be clear, negation
+             * is guaranteed to be meaningful in platform-neutral sense.
+             */
             *pr = -(int64_t)r;
         } else if (r == ABS_INT64_MIN) {
-            /* This never happens if INT64_MAX == ABS_INT64_MIN, e.g.
-             * on ones'-complement system. */
+            /*
+             * This never happens if INT64_MAX == ABS_INT64_MIN, e.g.
+             * on ones' complement system.
```

Comment:
Sorry, I don't think removing the hyphen is correct. In this case, we have a compound adjective, which should be written with a hyphen `ones'-complement system` (the same as is done with `platform-neutral` above). Whereas if we were to say something like `this doesn't happen in ones` complement` there wouldn't be a hyphen.